# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/win/src/remotesandboxbroker/remoteSandboxBroker.cpp
# Commit: 48eaa84c206d
# Full Hash: 48eaa84c206dbdda637c57b55ddbd07e975ba103
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-06-23 16:15:06
# Regressor Bug: 1637500
# File Overlap Count: 2
# Description:
#   Bug 1637500 - P2. Rename methods as they are not always dealing with "threads". r=froydnj
#   
#   Before P1, GetCurrentThreadSerialEventTarget would have always returned the same data as NS_GetCurrentThread, making the comment incorrect Now it will properly return the running TaskQueue if any.
#   
#   This change of name more clearly exposes what they are doing, as we aren't always dealing with threads directly; but a nsISerialEventTarget
# ==============================================================================

diff -r 00687d06a8a6 -r 48eaa84c206d security/sandbox/win/src/remotesandboxbroker/remoteSandboxBroker.cpp
--- a/security/sandbox/win/src/remotesandboxbroker/remoteSandboxBroker.cpp	Tue Jun 23 05:02:39 2020 +0000
+++ b/security/sandbox/win/src/remotesandboxbroker/remoteSandboxBroker.cpp	Tue Jun 23 05:05:36 2020 +0000
@@ -49,7 +49,7 @@
   // We should be on the IPC launch thread. We're shutdown on the IO thread,
   // so save a ref to the IPC launch thread here, so we can close the channel
   // on the IPC launch thread on shutdown.
-  mIPCLaunchThread = GetCurrentThreadEventTarget();
+  mIPCLaunchThread = GetCurrentEventTarget();
 
   mParameters.path() = nsDependentString(aPath);
   mParameters.args() = nsDependentString(aArguments);
@@ -78,7 +78,7 @@
   };
 
   mParent.Launch(mParameters.shareHandles())
-      ->Then(GetCurrentThreadSerialEventTarget(), __func__, std::move(resolve),
+      ->Then(GetCurrentSerialEventTarget(), __func__, std::move(resolve),
              std::move(reject));
 
   // Spin the event loop while the sandbox launcher process launches.